src/js/disability-benefits/containers/AppealStatusPage.jsx

```diff
@@ -23,13 +39,26 @@ class AppealStatusPage extends React.Component {
           </div>
         </div>
         <div className="row">
-          <div className="next-action">
-            next action explanation
-          </div>
-        </div>
-        <div className="row">
           <div className="small-12 usa-width-two-thirds medium-8 columns">
-            <Link to="appeals/learn-more">Learn more about the appeals process</Link>
+            <div className="row">
+              <div className="next-action">
+                <h4>{nextAction.title}</h4>
+                {nextAction.description}
+              </div>
+            </div>
+            <div className="row">
+              <div className="last-status">
+                <h4>{status.title}</h4>
+                <strong>{moment(lastEvent.date).format('MMM DD, YYYY')}</strong>
+                <p dangerouslySetInnerHTML={{ __html: status.description }}/>
```

Comment:
I think we should avoid using `dangerouslySetInnerHTML` where possible. I see you have some HTML in `appealStatusDescriptions`; you could just unstringify that and import React in that module and it would be fine.

With that, you could also remove the `eslint-disable` above, which seems to have never gotten enabled again.